### ISSUE TITLE
container does not work on ARM (raspberry pi)

### ISSUE THREAD

**Issue by kkmaslowski** [vanackej/risco-mqtt-local#55]
**Describe the bug**
Container image does not work on arm raspberry pi (arm).
When I downloaded image I saw message like above:
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

**Configuration**
 - Environment: Docker
 - Version: latest

**Logs**

```
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
```

**Comment by xyanide** [comment]
Have you tried the new beta 0.5 beta 2? This has been working on my raspberry pi for a couple of weeks now.

**Comment by kkmaslowski** [comment]
@xyanide no, I already moved from pi4 to x86. You have it running on 32 bit or 64 bit os? I tried to run it without success on latest Raspberry Pi OS 64 bit.

**Comment by xyanide** [comment]
> @xyanide no, I already moved from pi4 to x86. You have it running on 32 bit or 64 bit os? I tried to run it without success on latest Raspberry Pi OS 64 bit.

64 bit, but with the latest beta running via portainer on docker.
